swift/unowned-self-in-lazy-property.playground/Contents.swift

```diff
@@ -0,0 +1,44 @@
+/// This playground checks to make sure that you don't need to do [unowned self] in a lazy var
+/// closure that doesn't retain self.
+
+import Foundation
+
+class MyClass {
+  deinit {
+    print("MyClass dealloc")
+  }
+}
+
+class MyParentClass {
```

Comment:
Its strange to refer to this as a "Parent" class without it inheriting from `MyClass`.